service-commands/scripts/seed.js

```diff
@@ -0,0 +1,194 @@
+  .option('-p, --password <password>', 'password for user account', '')
+  .option('-n, --count <count>', 'number of random users to create', parseSeedActionRepeatCount, 1)
+  .option(
+    '-m, --metadata <metadata-object>',
+    'metadata to associate with user. Write this as a series of comma-separated key-values e.g. -m email=test@audius.co,password=2343,handle=christinus,is_verified=true',
+    parseMetadataIntoObject
+  )
+  .action(async opts => {
+    const { userAlias: alias, count, ...options } = opts.opts()
+    const createSingleUser = async (alias, options) => {
+      console.log(`Creating user with alias ${alias} and options: ${JSON.stringify(options)}`.info)
+      const seed = new SeedSession()
+      await seed.init()
+      await seed.createUser(alias, options)
+      console.log('Created user!'.success)
+    }
+    console.log(`Creating ${count} user(s)...`.info)
+    // unfortunately can't parallelize because of the way we need to get hedgehog entropy from same location in local storage...
+    for (const n of _.range(1, count + 1)) {
+      const createdUserAlias = n > 1 ? `${alias}-${n}` : alias
+      await createSingleUser(createdUserAlias, options)
+    }
+    console.log(`Created ${count} user(s) successfully!`.happy)
+    process.exit(0)
+  })
+
+program
+  .command('clear')
+  .description('Clears seed session and all locally cached data.')
+  .option('-f, --reset-state', 'whether to reset state on local DB', false)
+  .action(async opts => {
+    // TODO add reset state
+    console.log('Clearing seed session...'.info)
+    const seed = new SeedSession()
+    await seed.clearSession()
+    console.log('Seed session cleared.'.happy)
+    process.exit(0)
+  })
+
+program
+  .command('set-user')
+  .description(
+    'Set user as active for all following seed actions. Reference a user by either user ID or alias.'
+  )
+  .option('-a, --user-alias <alias>', 'alias of user to set as active', null)
+  .option('-u, --user-id <number>', 'ID of user to set as active', null)
+  .action(async opts => {
+    const { userAlias: alias, userId } = opts.opts()
+    if (!alias && !userId) {
+      throw new Error(
+        'Seed CLI: to set-user you must provide either a user ID or user alias of user to set as active.'
+      )
+    } else {
+      console.log(
+        `Setting active user for seed session to userId ${userId} / alias ${alias}.`.info
+      )
+      const seed = new SeedSession()
+      await seed.setUser({ alias, userId })
+      console.log(
+        `Active user for seed session set to userId ${userId} / alias ${alias}.`.success
+      )
+      process.exit(0)
+    }
+  })
+
+Object.entries(CLI_TO_COMMAND_MAP).forEach(
+  ([cliCommand, { api, description, method, params, onSuccess }]) => {
+    let configuredProgram = program
+      .command(cliCommand)
+      .description(description)
+    const userIdInLibsApiMethodParams = params.some(p => p.name === 'userId')
+    if (!userIdInLibsApiMethodParams) {
```

Comment:
Avoid declaring `--user-id` twice. We add it as a parameter by default for all seed actions in order to set the owner of the libs instance used to call APIs, but some libs API methods (such as `Playlist.createPlaylist` require userId to be passed in explicitly.